reana_client/cli/workflow.py

```diff
@@ -272,9 +276,14 @@ def workflow_create(ctx, file, name,
          'E.g. CACHE=off. (workflow engine - serial) '
          'E.g. --debug (workflow engine - cwl)',
 )
+@click.option(
+    '--follow', 'follow',
+    is_flag=True,
+    help='If set, follows the execution of the workflow until termination.',
+)
 @click.pass_context
 def workflow_start(ctx, workflow, access_token,
-                   parameters, options):  # noqa: D301
+                   parameters, options, follow):  # noqa: D301
```

Comment:
This is not related to the PR, but perhaps you can fix a cosmetic help issue alongside this PR?  The help for `start` does not display well currently:

```console
$ reana-client start --help  
Usage: reana-client start [OPTIONS]

  Start previously created workflow.

  The `start` command allows to start previously created workflow. The
  workflow execution can be further influenced by passing input prameters
  using `-p` or `--parameters` flag and by setting additional operational
  options using `-o` or `--options`.  The input parameters and operational
  options can be repetitive. For example, to disable caching for the Serial
  workflow engine, you can set `-o CACHE=off`.

  Examples: \n \t $ reana-client start -w myanalysis.42 -p sleeptime=10 \n
  \t $ reana-client start -w myanalysis.42 -p myparam1=myvalue1 -p
  myparam2=myvalue2 -o CACHE=off
```

Note `\n` and `\t` in the ouput.  

Should look like: 

```console
$ reana-client run --help        
Usage: reana-client run [OPTIONS] SOURCES

  Shortcut to create, upload, start a new workflow.

  The `run` command allows to create a new workflow, upload its input files
  and start it in one command.

  Examples:

       $ reana-client run -n myanalysis-test-small -p myparam=mysmallvalue

       $ reana-client run -n myanalysis-test-big -p myparam=mybigvalue

Options:
  -f, --file PATH          REANA specifications file describing the workflow
                           and context which REANA should execute.
  -n, --name TEXT          Name of the workflow.

```

Reply:
No problem, I found the issue.
I'll commit the changes now.